docs/source/pipeline.rst

```diff
@@ -0,0 +1,291 @@
+=========
+Pipelines
+=========
+
+Vertica Pipelines is an open source platform for managing data
+scientists’ machine learning pipelines. They are built on a
+human-readable data format: **YAML**.
+
+==============================
+Setting Up Your First Pipeline
+==============================
+
+Requirements
+~~~~~~~~~~~~
+| To begin, you must:
```

Comment:
Do not use "|" unnecessarily. This causes some consistency issues at times. 

The below produces bullets. So maybe you can try that:

To begin, you must:

- Have access to a machine that has Vertica installed
- Install Python on your machine
- Install Verticapy

NOTE: Do not forget to add a space before bullets.